src/main/java/frc/robot/Config.java

```diff
@@ -456,9 +456,9 @@ public static enum ArmSetPoints {
   //@todo: to be calibrated
   IDLE(61),
   INTAKE(-1.1),
-  SPEAKER_KICKBOT_SHOT(12.9),
+  SPEAKER_KICKBOT_SHOT(13.9),
   NO_INTAKE(2.2),
-  SPEAKER_VISION_SHOT(30.2),
+  SPEAKER_VISION_SHOT(30.2), // 31.2
```

Comment:
My impression with vision shot at the practice field looks good. May not need to be adjusted.